testcafe/tests/pages/drive-model-public.js

```diff
@@ -0,0 +1,118 @@
+import { Selector, t } from 'testcafe'
+import {
+  getElementWithTestId,
+  isExistingAndVisibile,
+  getPageUrl,
+  goBack,
+  getResponseStatusCode
+} from '../helpers/utils'
+import osHomedir from 'os-homedir'
```

Comment:
I think we should add this as a dependency? Although from the [npm page](https://www.npmjs.com/package/os-homedir):

> This is not needed anymore. Use `require('os').homedir()` instead.